README.md

```diff
@@ -57,8 +62,8 @@ $ yarn -v
 $ java -version
 ```
 
-Your Node.js version should be `10.15.0` or greater, your `yarn` version should
-be `1.0.0` or greater, and your `java` version should be `1.8.0` or greater.
+Your `node` version should be between `10.15.0` and `16.6.0`, your `yarn` version should
+be between `1.0.0` and `1.22.11`, and your `java` version should be `11.0` or greater.
```

Comment:
I would also capitalize java.

Reply:
I figured since yarn and java were in single quotes that these should refer to the binaries that the user executes, not necessarily the software's proper name. The documentation was inconsistent in this regard originally as it was Node.js `yarn` and `java`.

I went for consistency in binary executable names: `node`, `yarn`, `java`, but if you would rather I refer to them as Yarn, Node.js and Java, then I could that direction instead.